src/v/kafka/server/tests/client_quota_test_helpers.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2024 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+#include "base/seastarx.h"
+
+#include <seastar/core/smp.hh>
+
+#include <cstdint>
+
+inline uint64_t scaled(uint64_t val) {
```

Comment:
suggestion(non-blocking): a slightly more descriptive name might be a good idea here, just to keep the tests readable, e.g. `scale_to_smp_count` or something.

Reply:
I intentionally kept the name of this function short to prevent it from interfering with the formatting which I think is better for readability. It also forces readers to step into the function and read that comment, which I think provides useful context.

Reply:
I agree with a more specific name, but at least maybe at least not in the global scope?

Reply:
Fair enough, I've now moved it into the `kafka` namespace and renamed it to `scale_to_smp_count`